packages/plugin-page-view-tracking/src/typings/page-view-tracking.ts

```diff
@@ -0,0 +1,5 @@
+export interface PageTrackingBrowserOptions {
+  filter?: PageTrackingFilter;
```

Comment:
Depending on how we export this in `analytics-ma` we might want to give this a more descriptive name e.g. `pageTrackingFilter` so that it makes sense in the top-level config.

Do you plan to have the plugin config namespaced in the ma-client?
```
amplitude.init(apiKey, undefined, {
 pageTracking: { filter: 'onAttribution' }
}
```
or
```
amplitude.init(apiKey, undefined, {
  pageTrackingFilter: 'onAttribution'
}
```

I prefer the first one.

Reply:
It would be the first one. There are a few options:
```ts
amplitude.init(apiKey, undefined, {
 trackPageViews: { filter: 'onAttribution' }
}
```

```ts
amplitude.init(apiKey, undefined, {
 trackPageViews: { filter: () => true }
}
```

```ts
amplitude.init(apiKey, undefined, {
 trackPageViews: true
}
```

Currently we only have filter. It might have more options for `trackPageViews` in next a few iteration.